packages/relay/src/utils.ts

```diff
@@ -55,4 +58,55 @@ export class Utils {
         throw new Error(`Invalid OPERATOR_KEY_FORMAT provided: ${process.env.OPERATOR_KEY_FORMAT}`);
     }
   }
+
+  /**
+   * Generate logs bloom for synthetic transaction
+   * @param address
+   * @param topics
+   */
+  public static createSyntheticLogsBloom(address: string, topics: string[]): string {
```

Comment:
We can also do validation on the inputs here:
```typescript
  public static createSyntheticLogsBloom(address: string, topics: string[]): string {
    if (!address?.length) {
      throw new Error("Address must be a non-empty string");
    }
    if (!Array.isArray(topics) || topics.some(topic => typeof topic !== 'string')) {
      throw new Error("Topics must be an array of strings.");
    }
    ...
  }
```

Reply:
Or we can return an empty bloom in those cases, but I like the idea of having validations more because we can spot if there is a bug anywhere - unless it's expected that we receive invalid inputs here.